### ISSUE TITLE
Support filtering per URL across database

### ISSUE THREAD

**Issue by agrabeli** [ooni/explorer#133]
A feature that *many* people care about is being able to check the accessibility of a specific site globally (i.e. check in which countries a specific site is blocked).

The new UI of OONI Explorer creates the impression that you can easily filter the measurements to get this sort of information by adding the URL of interest in the input section, and then clicking "Filter Results" (without specifying the country) to check the accessibility of the selected URL across the whole OONI database. However, this doesn't work.

I imagine we need to make some improvements to the pipeline to support this?

**Comment by hellais** [comment]
> clicking "Filter Results" (without specifying the country) to check the accessibility of the selected URL across the whole OONI database. However, this doesn't work

Can you clarify what you mean exactly when you say "this doesn't work"?

If you could provide step by step instructions on how to reproduce the issue that would be great as well.

In the meantime I am removing the bug label as there is not enough information for this to be investigated.

**Comment by agrabeli** [comment]
@hellais @sarathms When I opened this ticket, I would type in a URL in the Input section of the Search page, and then click "Anomalies" and "Filter Results". At the time, this wouldn't render any results, which is why I said that it "didn't work". 

However, I am now able to filter measurements based on a single URL, and it works beautifully! This is so exciting! :)

I'll close this ticket (as either the issue has since been resolved, or it was an issue on my network).
